doc/api/http.md

```diff
@@ -1880,6 +1880,16 @@ const req = http.request({
 });
 ```
 
+### `message.connection`
+<!-- YAML
+added: v0.11.1
```

Comment:
I don't think this is correct. Just judging by documentation alone, `.connection` was documented prior to the `http` docs rewrite in v0.9.9, *at least* as far back as v0.1.x.

Reply:
This was moved to its own file in https://github.com/nodejs/node/commit/5909a9c9bddddaab91693d18a8840d8b300bbc28#diff-f7d8a311cc7ac30522e414465bda2229R47, so the blame in this file goes as far as that, but I traced it a bit further and found that it was added in https://github.com/nodejs/node/commit/3546106c43c08aed0f3e45f81f82496c9c51f6b9. I've made the change.